CHANGELOG.md

```diff
@@ -0,0 +1 @@
+- Add support for more regions and memory for v2 functions (#1037).
```

Comment:
Should we add a changelog entry for this PR too?

Reply:
I chose not to since this change is refactoring non-user facing APIs. Let me know if you disagree.